src/components/StatusBar/StatusBar.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import css from "./StatusBar.scss";
+import { getAssetInfo, makeEtherscanTokenURL } from "./../../utils";
+import { isERC721Token, getTokenOwner } from "./../../services/token";
+import { getLogger } from "./../../utils/logger";
+const { trace, error } = getLogger("Component:DetailedCertificateVerifyBlock");
+
+interface StatusBarProps {
+  document: object;
+}
+
+const StatusBar = ({ document }: StatusBarProps) => {
+  const { tokenRegistry: registryAddress, tokenId } = getAssetInfo(document);
+
+  const { holderAddress, beneficiaryAddress } = useSelector((state: any) => ({
+    holderAddress: state.token.holderAddress,
+    beneficiaryAddress: state.token.beneficiaryAddress
+  }));
+
+  const [tokenError, setError] = useState(null);
+  const [tokenOwner, setTokenOwner] = useState("");
+  const isToken = isERC721Token(document);
+  useEffect(() => {
+    async function fetchTokenOwner() {
+      try {
+        const owner = await getTokenOwner({ document });
+        trace(`Token Owner: ${owner}`);
+        setTokenOwner(owner);
+      } catch (e) {
+        error(`error in fetching token owner: ${JSON.stringify(e)}`);
+        setError(e.message);
+      }
+    }
+    if (isToken) fetchTokenOwner();
+  }, [document, isToken]);
+
+  return (
+    <>
+      {isToken && (
```

Comment:
can move this check to in function and return null. i.e if(!isToken) return null;